<launch>

	<node pkg="gnss_dummy" name="GNSS" type="gnss_dummy_node" output="screen" />
	<test pkg="logger" test-name="logger_text" type="test_logger_text_node"/>
	
</launch>
